webclient/js/elems/transactant.js

```diff
@@ -61,115 +87,209 @@ window.getTransactantInitRecord = function (BID, BUD) {
+    ];
+
+    app.transactantTabs = [
+        {id: 'tab1', caption: app.sTransactant},
+        {id: 'tab2', caption: app.sUser},
+        {id: 'tab3', caption: app.sPayor},
+        {id: 'tab4', caption: app.sProspect}
+    ];
+
+    //------------------------------------------------------------------------
+    //          transactantsGrid
+    //------------------------------------------------------------------------
+    $().w2grid({
+        name: 'transactantsGrid',
+        url: '/v1/transactants',
+        multiSelect: false,
+        show: {
+            header: false,
+            toolbar: true,
+            footer: true,
+            toolbarAdd: true,
+            lineNumbers: false,
+            selectColumn: false,
+            expandColumn: false
         },
-        {field: 'CompanyName',  caption: "Company Name",  size: '125px', sortable: true, hidden: false,
-            render: function (record) {
-                var s = '';
-                if (typeof record === "undefined") {
-                    return;
-                }
-                if (record.IsCompany) {
-                    s += '<span style="color:#999;font-size:16px"><i class="far fa-handshake" aria-hidden="true"></i></span>';
+        columns: [
+            {field: 'TCID',         caption: "TCID",          size: '50px',  sortable: true, style: 'text-align: right', hidden: false},
+            {field: 'FirstName',    caption: "First Name",    size: '125px', sortable: true, hidden: false},
+            {field: 'MiddleName',   caption: "Middle Name",   size: '20px',  sortable: true, hidden: true},
+            {field: 'LastName',     caption: "Last Name",     size: '125px', sortable: true, hidden: false,
+                render: function (record) {
+                    var s = '';
+                    if (typeof record === "undefined") {
+                        return;
+                    }
+                    if (!record.IsCompany) {
+                        s += '<span style="color:#999;font-size:16px"><i class="far fa-handshake" aria-hidden="true"></i></span>';
+                    }
+                    return s + ' ' + record.LastName;
                 }
-                return s + ' ' + record.CompanyName;
-            }
-        },
-        {field: 'PrimaryEmail', caption: "Primary Email", size: '175px', sortable: true, hidden: false},
-        {field: 'CellPhone',    caption: "Cell Phone",    size: '100px', sortable: true, hidden: false},
-        {field: 'WorkPhone',    caption: "Work Phone",    size: '100px', sortable: true, hidden: false},
-    ],
-    onRefresh: function(event) {
-        event.onComplete = function() {
-            if (app.active_grid == this.name) {
-                if (app.new_form_rec) {
-                    this.selectNone();
+            },
+            {field: 'CompanyName',  caption: "Company Name",  size: '125px', sortable: true, hidden: false,
```

Comment:
shouldn't this be EmployerName?

Reply:
I'm not sure about this.

Reply:
Yes, you should look for this in source tree entirely and replace it with EmployerName, If you're changing field from any db table then please look for that fieldname in entire source tree to replace with new one!

Reply:
We might require to clarify about it. User table has CompanyName and Payor table has EmployerName two fields. Which one we have to keep?

Reply:
yes, so please clarify that with Steve. I'm not sure about this too! We should keep only one of both throughout the application. Otherwise, it will create confusion!

Reply:
Yeah, I asked Steve over slack. Let's wait for his reply in this case.